packages/headless/src/features/commerce/facets/numeric-facet/numeric-facet-actions.ts

```diff
@@ -69,17 +80,28 @@ export const updateNumericFacetValues = createAction(
   }
 );
 
+export type UpdateManualNumericFacetRangePayload = {
+  /**
+   * The unique identifier of the facet (e.g., `"1"`).
+   */
+  facetId: string;
+} & NumericRangeRequest;
+
 export const updateManualNumericFacetRange = createAction(
   'commerce/facets/numericFacet/updateManualRange',
   (payload: UpdateManualNumericFacetRangePayload) =>
     validatePayloadAndThrow(payload, {
       facetId: requiredNonEmptyString,
-      start: new NumberValue({required: true, min: 0}),
-      end: new NumberValue({required: true, min: 0}),
-      endInclusive: new BooleanValue({required: true}),
-      state: new StringValue<'idle' | 'selected' | 'excluded'>({
-        required: true,
-        constrainTo: ['idle', 'selected', 'excluded'],
-      }),
+      ...numericFacetValueDefinition,
     })
 );
+
+const numericFacetValueDefinition = {
+  state: new StringValue<'idle' | 'selected' | 'excluded'>({
+    required: true,
+    constrainTo: ['idle', 'selected', 'excluded'],
+  }),
+  start: new NumberValue({required: true, min: 0}),
+  end: new NumberValue({required: true, min: 0}),
```

Comment:
Those are more restrictive validation rules than [those we are currently enforcing](https://github.com/coveo/ui-kit/blob/904df53b166d4e4b47b86cb429c270a1a11f075e/packages/headless/src/features/facets/range-facets/generic/range-facet-validate-payload.ts#L6-L12).

I'm not sure about enforcing `min: 0` on `start` and `end`... could a numeric facet value theoretically be negative?

However, constraining the strings that can be passed in `state` to the actual values that are accepted by the API is IMO better, because it would just end up failing further down the line if any other value was passed.

Reply:
I agree with yassine

Reply:
I even found this discuss post about negative numerical values. https://discuss.coveo.com/t/input-boxes-of-atomic-numeric-facet/7767

Reply:
💯 

I'll remove the `min: 0` requirement in the validation for both `start` and `end`.